app/controllers/data_exports_controller.rb

```diff
@@ -40,12 +39,13 @@ def create
     @data_export = DataExports::CreateService.new(current_user, data_export_params).execute
 
     if @data_export.valid?
-      flash[:success] = t('.success')
+      flash[:success] = t('.success', name: @data_export.name || @data_export.id)
       redirect_to data_exports_path
```

Comment:
**nitpick (blocking):** Can this be changed to redirect to the newly created data export show page

Reply:
Changed in cfedc0193d0aac85151df7f747ea03b11224bb31